packages/server/lib/config.ts

```diff
@@ -215,6 +214,19 @@ export function mergeDefaults (
     throw makeConfigError(errors.get(err, ...args))
   }, testingType)
 
+  // TODO: testIsolation should equal 'strict' by default when experimentalSessionAndOrigin=true
```

Comment:
Will we eventually apply `strict` to everything? My understanding is we have `strict` for `cy.origin` to make it more reliable (clean slate). Since this seems ideal for all tests, is the intention to make this the default in Cypress 11?

Reply:
`cy.origin` is tied to and dependent on the sessions feature because it enables clearing cookies/localstorage/etc for all origins, whereas historically it's only been cleared for the primary origin. Test isolation is a separate feature that, as you intuited, should benefit all e2e tests regardless of whether or not they use `cy.origin`.